src/common/flow_lsp_conversions.ml

```diff
@@ -18,17 +18,21 @@ let flow_completion_to_lsp
     let params = Core_list.map ~f:(fun p -> p.param_name ^ ": " ^ p.param_ty) params in
     "(" ^ (String.concat ", " params) ^ ")"
   in
-  let itemType, inlineDetail, detail = match item.func_details with
+  let flow_params_to_lsp_snippet name params =
+    let params = Core_list.mapi ~f:(fun i p -> "${" ^ string_of_int (i +1 ) ^ ":" ^ p.param_name ^ "}") params in
+      name ^ "(" ^ (String.concat ", " params) ^ ")"
```

Comment:
nit: indentation is off from the previous line

Reply:
fixed